README.md

```diff
@@ -200,6 +200,17 @@ await flatDirectory.download(key, {
 });
 ```
 
+Note: If you only need the download function, you can create a FlatDirector using the following pattern.
```

Comment:
1. FlatDirector => FlatDirectory
2. This part can be moved to the create part

Reply:
fix